gnmpagerduty/__init__.py

```diff
@@ -0,0 +1,8 @@
+from plugin import *
+from tasks import check_storage
+
+__version__ = 'DISPLAY_VERSION'
+__production__ = False
+
+check_storage.delay()
```

Comment:
I think I am already using that though maybe the syntax is wrong. This is the line: -

@periodic_task(run_every=(crontab(minute='*/10')), name="check_storage", ignore_result=True)

Without the call to check_storage this does nothing, hence the call to check_storage